### DISCUSSION TITLE
Output boundary heat flux

### DISCUSSION THREAD

**Discussion by lucuo** [idaholab/moose#27947] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [x] I have searched the MOOSE website and the documentation does not answer my question
- [x] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

How to output the boundary heat flux in the process of 3D heat transfer calculation. It's best to keep it in Lagrange auxvariable. thanks a lot

**Comment by GiudGiud** [comment]
Hello

This should work unless you used an exotic FE discretization
https://mooseframework.inl.gov/source/postprocessors/SideDiffusiveFluxIntegral.html

**Comment by lucuo** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，已收到你的形势与政策作业，祝你天天愉快~

**Reply by GiudGiud** [reply]
你好
